### ISSUE TITLE
Doesn't work with recent search_api_solr versions

### ISSUE THREAD

**Issue by ccato** [dcameron/search_api_solr_datasource#11]
I haven't narrowed it down to the exact commit that broke it but it's broken from 1.0 and onwards. Adding a field in Views just gives me stuff like "[fieldname] indexed as string" and so on. I'll dig deeper next week.

**Comment by DavidACameron-USDA** [comment]
Ok, thank you for letting me know.  I'll try to work on this over the weekend or next week.

**Comment by ccato** [comment]
Yeah, no problems. I saw there was another (closed) bug that was related to this but there was some mention of ss_language being key to getting it to work. In my case there is no ss_language field in my index. The index doesn't contain anything like that.

**Comment by ccato** [comment]
After a bisect it seems that this is the commit that broke the integration. Hope it's helpful.

commit 20208ebce4aad8cdb7b1498d1cd1169fc310a35c
Date:   Fri Jun 9 10:50:18 2017 +0200
Issue #2884494 by mkalkbrenner, ayalon: Don't always return all fields from solr index
